### PR TITLE
Merge sfranklin/featured speakers to staging

### PR DESCRIPTION
**Pull request by simeonf**

### CODE REVIEW

File: featured_speakers/templatetags/featured_tags.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from featured_speakers.models import FeaturedSpeaker
+
+register = template.Library()
+
+
+@register.inclusion_tag('featured_speakers/list.html', takes_context=True)
+def featured_speakers(context):
```
Comment (epsy) [inline featured_speakers/templatetags/featured_tags.py]: please match file title and tag title

File: featured_speakers/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (epsy) [inline featured_speakers/tests.py]: delete useless file

File: featured_speakers/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (epsy) [inline featured_speakers/views.py]: delete useless file

**Comment by simeonf** [comment]
Never mind - looks like Daniel got this already.

File: featured_speakers/templates/featured_speakers/list.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+		  <h1 class="text-center">Some of our speakers</h1>
+		</div>
+	  </div>
+	  <div class="row speakers-row">
+      {% for featured in speakers %}
+		<div class="col-md-3 col-sm-6 speaker-column">
+		  <div class="speaker">
+			<div class="image-holder">
+              <img width="240" class="background-image" src="{% speaker_url_with_fallback featured.speaker 240 %}" alt="{{ featured.speaker.name }}">
+			  {# <img class="background-image" alt="Speaker" src="{{ MEDIA_URL }}{{ featured.speaker.photo.name }}"> #}
```
Comment (epsy) [inline featured_speakers/templates/featured_speakers/list.html]: delete commented out code

File: featured_speakers/templates/featured_speakers/list.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+			  <div class="hover-state text-center preserve3d">
+				<div class="social-links vertical-align">
+                  {% if featured.speaker.twitter_username %}
+				  <a href="http://twitter.com/{{ featured.speaker.twitter_username }}"><i class="icon social_twitter"></i></a>
+                  {% endif %}
+				</div>
+			  </div>
+			</div>
+			<span class="speaker-name">{{ featured.speaker.name }}</span>
+			{{ featured.speaker.annotation|safe }}
```
Comment (epsy) [inline featured_speakers/templates/featured_speakers/list.html]: The way I understand it, annotation shouldn't be visible to anyone but staff

Comment (epsy) [reply featured_speakers/templates/featured_speakers/list.html]: Add a field in 'featured' if you want this

**Review by epsy — COMMENTED** [review]
The images are very inconsistent, see staging

**Pull request closed without merging**